curvlinops/fisher.py

```diff
@@ -253,6 +244,7 @@ def sample_grad_output(self, output: Tensor, num_samples: int) -> Tensor:
             output: model prediction ``f`` for multiple data with batch axis as
                 0th dimension.
             num_samples: Number of samples to draw.
+            y: Ground truth.
```

Comment:
I think I would prefer another term here, something like "empirical label" or "label from data". "Ground truth" implies to me that this is what we are trying to approximate or what would be the ideal object to use.

Reply:
Agree.